src/components/Tabbar/index.js

```diff
@@ -1,44 +1,45 @@
 import React from 'react';
-import {View, TouchableOpacity, StyleSheet} from 'react-native';
-import Animated from 'react-native-reanimated';
-import {fonts} from '../../utils/fonts';
+import {View, TouchableOpacity, StyleSheet, Text} from 'react-native';
+import {fonts, normalizeFontSize} from '../../utils/fonts';
 import {COLORS} from '../../utils/theme';
+import dimen from '../../utils/dimen';
 
 const S = StyleSheet.create({
   container: {
-    paddingHorizontal: 22,
-    paddingVertical: 20,
+    paddingHorizontal: dimen.normalizeDimen(22),
+    paddingVertical: dimen.normalizeDimen(20),
     display: 'flex',
     flexDirection: 'column'
   },
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The use of `display: 'flex'` and `flexDirection: 'column'` is redundant here as these are default styles in React Native. Removing these lines will simplify the code without changing its behavior.
```diff
-    container: {
-     paddingHorizontal: dimen.normalizeDimen(22),
-     paddingVertical: dimen.normalizeDimen(20),
-      display: 'flex',
-      flexDirection: 'column'
+    container: {
+     paddingHorizontal: dimen.normalizeDimen(22),
+     paddingVertical: dimen.normalizeDimen(20)
    },
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->